.github/workflows/nightly-test-run.yml

```diff
@@ -71,6 +76,11 @@ jobs:
           PYFLUENT_LAUNCH_CONTAINER: 1
           FLUENT_IMAGE_TAG: v23.1.0
 
+      - name: Print 23.1 Fluent version info
+        run: |
+          cat src/ansys/fluent/core/fluent_version_231.py
+          python -c "from ansys.fluent.core.solver.settings_231 import SHASH; print(f'SETTINGS_HASH = {SHASH}')"
```

Comment:
When the settings API is changed in the Fluent side, this hash value will change. Nightly failures can be corresponded with the possible settings API change by tracking this output.

Reply:
Thanks Mainak for accommodating this request.